src/GovUk.Education.ExploreEducationStatistics.Data.Processor.Model/CancelImportMessage.cs

```diff
@@ -4,8 +4,11 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Model
 {
     public class CancelImportMessage
     {
-        public Guid ReleaseId { get; set; }
-        
-        public string DataFileName { get; set; }
+        public Guid Id { get; set; }
```

Comment:
There's not really anything that we've done specially to use JSON here, that's just the standard used for messaging queues for deserializing the message payloads containing complex types. The Azure function recognises the JSON and tries to deserialize it into the type mentioned in the function body. If it can deserialize it successfully it will trigger the function. If not it will remain on the queue for another function to possibly consume it in future. 

Yes the Azure Functions could be changed here to not use the message classes here, as functions can also be invoked from string or byte. I'm not sure about `Guid` so we'd have to investigate that. It's not a type mentioned in the docs.

I think here the message classes complement the contract between the client and the functions, by defining the type of message expected, even though they only contain one (or in some cases no) fields.